sql/core/src/main/scala/org/apache/spark/sql/execution/basicOperators.scala

```diff
@@ -102,6 +106,8 @@ case class Limit(limit: Int, child: SparkPlan)
   override def output: Seq[Attribute] = child.output
   override def outputPartitioning: Partitioning = SinglePartition
 
+  override def outputOrdering: Seq[SortOrder] = child.outputOrdering
```

Comment:
I am not sure it is correct. We are merging rows from multiple partitions to a single partition and `outputOrdering` only guarantee the row ordering within a single partition. Seems without merge sort, we cannot use `child.outputOrdering`. How about we just remove it for now?